### PR TITLE
Correct bug from postgres comment using quote or caps

### PR DESCRIPTION
**Pull request by Ailurupoda**
## Description
Add the possibility to comment a postgreSQL table using  quote and other special characters.

#34362

### CODE REVIEW

File: python/plugins/db_manager/db_plugins/postgis/connector.py [hunk 1]
```diff
@@ -878,7 +878,7 @@ def commentTable(self, schema, tablename, comment=None):
         if comment is None:
             self._execute(None, 'COMMENT ON TABLE "{0}"."{1}" IS NULL;'.format(schema, tablename))
         else:
-            self._execute(None, 'COMMENT ON TABLE "{0}"."{1}" IS E\'{2}\';'.format(schema, tablename, comment))
+            self._execute(None, 'COMMENT ON TABLE "{0}"."{1}" IS $${2}$$;'.format(schema, tablename, comment))
```
Comment (nyalldawson) [inline python/plugins/db_manager/db_plugins/postgis/connector.py]:
```suggestion
            self._execute(None, 'COMMENT ON TABLE "{0}"."{1}" IS $escape${2}$escape$;'.format(schema, tablename, comment))
```

Just to play it super-safe?

Comment (Ailurupoda) [reply python/plugins/db_manager/db_plugins/postgis/connector.py]: Seems good to me. It's at least cleaner :)

**Comment by Ailurupoda** [comment]
How to solve the conflicts with qgis.QGIS Azure Pipelines ? i dont' really understand how this works ..

**Comment by nyalldawson** [comment]
@Ailurupoda it was an unrelated test failure

**Pull request merged**